rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1090,6 +1097,7 @@ def __init__(
         self._update_metrics_to_log()
 
         self.all_labels_embed = None  # needed for efficient prediction
+        self._prepare_layers()
```

Comment:
this is called in super class

Reply:
I changed it because it caused the `you should always start with super().__init__()` error. And there is other no way around it.

Reply:
did you remove it from `TransformerRasaModel`?

Reply:
Yes, and added  into `TED` and `DIET`